upup/models/cloudup/resources/addons/storage-aws.addons.k8s.io/v1.15.0.yaml

```diff
@@ -15,7 +15,7 @@ kind: StorageClass
 metadata:
   name: gp2
   annotations:
-    storageclass.beta.kubernetes.io/is-default-class: "false"
+    storageclass.kubernetes.io/is-default-class: "false"
```

Comment:
I may be mistaken but I think the reason I left the old annotations was because k8s was reading both still? As in, not setting `storageclass.beta.kubernetes.io/is-default-class: "false"` on the `gp2` class and trying to just set `storageclass.kubernetes.io/is-default-class: "false"` caused the same issues with which one ended up being the default. Would have to test to confirm though

Reply:
Apply should remove all unset annotations.